### PR TITLE
Update project examples

### LINKED ISSUE #77: Examples scripts need to match latest network operator release features
- [ ] The examples CRs file still uses the old MOFED format, it need to be updated.
- [ ] Also the secondary network example scripts are still deploying the multus and whereabout CNIs, which are being deployed by the operator, this needs to be updated to avoid having two multus and whereabout pods deployed on the nodes. Instead a macvlan CR needs to be created.

### PR DESCRIPTION
**Pull request by Mmduh-483**
Removed unneeded examples scripts and deployment files for deploying secondary network components
Fixes #77

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -9,7 +9,10 @@ network-operator, at this stage, deploys and configures the follwoing components
 * [Mellanox OFED](https://www.mellanox.com/products/infiniband-drivers/linux/mlnx_ofed) driver container
 * [RDMA device plugin](https://github.com/Mellanox/k8s-rdma-shared-dev-plugin)
 * [NVIDIA peer memory client](https://github.com/Mellanox/nv_peer_memory) driver container
-
+* SecondaryNetwork`: Specifies components to deploy in order to facilitate a secondary network in Kubernetes. It consists of the following optionally deployed components:
+    - [Multus-CNI](https://github.com/intel/multus-cni): Delegate CNI plugin to support secondary networks in Kubernetes
+    - CNI plugins: Currently only [containernetworking-plugins](https://github.com/containernetworking/plugins) is supported
+    - IPAM CNI: Currently only [Whereabout IPAM CNI](https://github.com/dougbtv/whereabouts-cni) is supported
```
Comment (adrianchiris) [inline example/README.md:15]: Lines 23-24 should be removed as we now support deploying secondary network.

File: example/README.md [hunk 2]
```diff
@@ -25,15 +28,9 @@ This example aims to address the points above in a basic manner.
...
-* `deploy-rdma-net.sh`: Deploy secondary network for RDMA traffic with static IPAM.
-* `deploy-rdma-net-ipam.sh`: Deploy secondary network for RDMA traffic with cluster IPAM
-([wherabouts](https://github.com/openshift/whereabouts-cni)).
 * `delete-operator.sh`: Delete network operator and related resources.
-* `delete-rdma-net.sh`: Delete secondary network for RDMA traffic with static IPAM.
-* `delete-rdma-net-ipam.sh`: Delete secondary network for RDMA traffic with cluster IPAM
-([wherabouts](https://github.com/openshift/whereabouts-cni)).
 * `deploy`: network-operator related deployment files
-* `networking`: kubernetes secondary network and IPAM related deployment files 
+* `networking`: kubernetes secondary network and IPAM related deployment files for hostdev
```
Comment (adrianchiris) [inline example/README.md:33]: how do i deploy macvlan network ? i see you deleted deploy-rdma-net and deploy-rdma-net-ipam ?

Comment (adrianchiris) [reply example/README.md:33]: perhaps use the macvlannetwork CR and have two files, one for macvlan with wherabouts and one without.

Comment (Mmduh-483) [reply example/README.md:33]: macvlan can be deployed from the example/deply/crds example

Comment (Mmduh-483) [reply example/README.md:31]: Added exmaple for both cases

File: example/networking/mellanox.com_v1alpha1_macvlannetwork_ipam-whereabouts_cr.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@
 apiVersion: mellanox.com/v1alpha1
 kind: MacvlanNetwork
 metadata:
-  name: example-macvlannetwork
+  name: example-macvlannetwork-ipam-whereabouts
 spec:
   networkNamespace: "default"
   master: "ens2f0"
```
Comment (adrianchiris) [inline example/networking/mellanox.com_v1alpha1_macvlannetwork_ipam-whereabouts_cr.yaml:20]: can you set mtu to 9000 ?

Comment (adrianchiris) [reply example/networking/mellanox.com_v1alpha1_macvlannetwork_ipam-whereabouts_cr.yaml:20]: https://github.com/Mellanox/network-operator/blob/81f6a2260712389e99086d40ccdae7cbdef625ef/example/networking/rdma-net-cr-whereabouts-ipam.yml#L29

File: example/networking/mellanox.com_v1alpha1_macvlannetwork_ipam-whereabouts_cr.yaml [hunk 4]
```diff
@@ -14,7 +14,7 @@
 apiVersion: mellanox.com/v1alpha1
 kind: MacvlanNetwork
 metadata:
-  name: example-macvlannetwork
+  name: example-macvlannetwork-ipam-whereabouts
```
Comment (adrianchiris) [inline example/networking/mellanox.com_v1alpha1_macvlannetwork_ipam-whereabouts_cr.yaml:17]: pod example need to be updated with the new network name or better yet just keep the original name for macvlan network both with and without ipam

**Review by adrianchiris — CHANGES_REQUESTED** [review]

File: example/delete-rdma-net-ipam.sh [hunk 5]
```diff
@@ -16,6 +16,3 @@
 echo "Deleting Secondary Network with Whereabouts IPAM"
 echo "################################################"
 kubectl delete -f networking/rdma-net-cr-whereabouts-ipam.yml
-kubectl delete -f networking/whareabouts-daemonset-install.yaml
```
Comment (adrianchiris) [inline example/delete-rdma-net-ipam.sh:19]: this should be deleted as well

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**